src/components/tx-flow/common/TxLayout.tsx

```diff
@@ -32,7 +33,8 @@ const TxLayout = ({ title, children, step = 0, txSummary, onBack }: TxLayoutProp
             <Grid item xs={7} component={Paper}>
               <ProgressBar value={progress} />
 
-              <Box display="flex" justifyContent="flex-end" py={2} px={3}>
+              <Box display="flex" justifyContent="space-between" py={2} px={3}>
+                <span>{subtitle}</span>
```

Comment:
```suggestion
                {subtitle && <span>{subtitle}</span>}
```

Reply:
Not sure if this even needs be wrapped in a `span`, e.g.

```suggestion
                {subtitle}
```

Reply:
I left the span in so the txNonce is moved to the right side. We could also conditionally change the flex property. What do you prefer?